pyvista/plotting/renderer.py

```diff
@@ -355,15 +358,15 @@ def camera_position(self, camera_location):
 
         elif isinstance(camera_location[0], (int, float)):
             if len(camera_location) != 3:
-                raise pyvista.core.errors.InvalidCameraError
+                raise InvalidCameraError
```

Comment:
For another day: these messageless exceptions are weird.

Reply:
```suggestion
                raise InvalidCameraError('camera location vector's dimension is not equal to 3')
```

Reply:
Thanks @tkoyama010 but there's a bunch of others in this module which is why I didn't post a suggestion here.

Reply:
Syntax isn't right above. Let's resolve this in a follow up PR and do them all at once

Reply:
Actually, I was almost done writing an issue for this when I realised that probably all of these are custom exception types _with a default message_. And raising just the type implicitly instantiates the type, so this works. We probably have nothing to do here. Sorry for the noise.

In case you want to look at it, here's what made the penny drop for me:
```
$ git grep '^ *raise [^(]\+$'
pyvista/core/composite.py:                    raise err
pyvista/core/composite.py:        raise StopIteration
pyvista/core/composite.py:                            raise err
pyvista/core/dataset.py:            raise RuntimeError  # this should never be reached with err=True
pyvista/core/filters/poly_data.py:            raise NotAllTrianglesError
pyvista/core/pointset.py:        raise PointSetCellOperationError
pyvista/core/pointset.py:        raise PointSetCellOperationError
pyvista/core/pointset.py:        raise PointSetCellOperationError
pyvista/core/pointset.py:        raise PointSetCellOperationError
pyvista/core/pointset.py:        raise PointSetDimensionReductionError
pyvista/core/pointset.py:        raise PointSetDimensionReductionError
pyvista/core/pointset.py:        raise PointSetDimensionReductionError
pyvista/core/pointset.py:        raise PointSetDimensionReductionError
pyvista/core/pointset.py:        raise PointSetDimensionReductionError
pyvista/core/pointset.py:        raise PointSetCellOperationError
pyvista/core/pointset.py:        raise PointSetCellOperationError
pyvista/core/pointset.py:        raise PointSetCellOperationError
pyvista/core/pointset.py:        raise PointSetCellOperationError
pyvista/examples/downloads.py:                raise OSError
pyvista/plotting/plotting.py:                raise KeyboardInterrupt
pyvista/plotting/renderer.py:                raise pyvista.core.errors.InvalidCameraError
pyvista/plotting/renderer.py:                    raise pyvista.core.errors.InvalidCameraError
pyvista/plotting/renderer.py:                    raise pyvista.core.errors.InvalidCameraError
pyvista/utilities/algorithms.py:            raise e
pyvista/utilities/algorithms.py:            raise e
pyvista/utilities/algorithms.py:            raise e
pyvista/utilities/algorithms.py:            raise e
pyvista/utilities/reader.py:            raise NotImplementedError
pyvista/utilities/reader.py:                raise err
tests/plotting/test_plotting.py:            raise err
tests/tinypages/plot_polygon.py:    raise NotImplementedError
```
The only sus one in there is the `raise RuntimeError` which should probably be an assertion error. Not worth doing more work for, probably.

Consider this resolved indeed.